include/rocksdb/env.h

```diff
@@ -171,7 +171,7 @@ class Env : public Customizable {
   Env(const Env&) = delete;
   void operator=(const Env&) = delete;
 
-  virtual ~Env();
+  virtual ~Env() override;
```

Comment:
According to `-modernize-use-override`(https://clang.llvm.org/extra/clang-tidy/checks/modernize-use-override.html), we should remove `virtual` and keep `override`.

Reply:
TIL :)